### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pleocavee**
Minor fixes to deal with deployment identifier and site name command, and to millisecond time offset which isn't needed with the new sleepmcu function (which was causing an issue in the sdcard log)

### CODE REVIEW

File: src/datalogger.cpp [hunk 1]
```diff
@@ -59,7 +59,8 @@ void Datalogger::sleepMCU(int milliseconds)
 
     reenableAllInterrupts(iser1, iser2, iser3);
 
-    offsetMillis += milliseconds; // systick was stopped, add slept milliseconds
+    //offsetMillis += milliseconds; // systick was stopped, add slept milliseconds
+
```
Comment (ZavenArra) [inline src/datalogger.cpp:63]: Why is this removed?  Since the systick was stopped during sleeping, don't we need to add these milliseconds back in?

File: src/sensors/base/analog_protocol_driver.h [hunk 2]
```diff
@@ -2,6 +2,9 @@
 *  Base class for sensor drivers using the Analog protocol
 */
 
+#ifndef WATERBEAR_ANALOG_PROTOCOL_DRIVER
+#define WATERBEAR_ANALOG_PROTOCOL_DRIVER
+
```
Comment (ZavenArra) [inline src/sensors/base/analog_protocol_driver.h:7]: Thanks!

File: src/sensors/drivers/generic_analog.cpp [hunk 3]
```diff
@@ -187,6 +187,8 @@ void GenericAnalogDriver::takeCalibrationBurstMeasurement()
       externalADC->convertEnabledChannels();
     }
     takeMeasurement();
+    // TODO: check for 0 values, report them, and either skip them for calibration, or halt calibration and notify that board may have electrical issues?
+    // Thoughts: reasons for 0 values? electrical issue, power failure, or are there scenarios where 0's are acceptable but still need to be skipped?
```
Comment (ZavenArra) [inline src/sensors/drivers/generic_analog.cpp:191]: A value that's at the absolute edge of the range of the sensor is always very very suspicious.  I don't think they should be accepted during calibration - though we also have to make sure that if we get only 0's we aren't eternally stuck in calibration mode.

**Review by ZavenArra — CHANGES_REQUESTED** [review]

File: src/datalogger.cpp (hunk 1 above)
Comment (pleocavee) [reply src/datalogger.cpp:63]: Ah, I'm not quite sure, but the line was causing issues. I thought maybe the sleepMCU function was being called in a different location or was affecting something else. It's not affecting the time calculation in the current code though. Maybe there's something different in the code that writes all of the output at once?

Comment (ZavenArra) [reply src/datalogger.cpp]: it needs to be a subtraction, I had misunderstood the logic here. Applied a fix and can merge now.
